rpc/src/v1/tests/helpers/signature.rs

```diff
@@ -0,0 +1,57 @@
+use std::sync::Arc;
```

Comment:
New files need a preamble, but I think it would be much better to add the tests in the same file as the function-under-test. Just add `#[cfg(test)] mod tests {` and put the content there.